src/api/controllers/user.js

```diff
@@ -1,111 +1,70 @@
-import User from '../models/user.js';
-import getAutoSuggestUsers from '../util/getAutoSuggestUsers.js';
+import User from '../../models/user.js';
+import getAutoSuggestUsers from '../../utils/getAutoSuggestUsers.js';
+import UserService from '../../services/userService.js';
 
-export const getAllUsers = (req, res, next) => {
+export const getAllUsers = async (req, res, next) => {
   try {
-    const allUsers = User.fetchAll();
-    if (allUsers.length > 0) {
-      return res.status(200).json({
-        message: 'Fetched users successfully!',
-        totalUsers: allUsers,
-      });
-    }
-    res.status(200).json({ message: 'No users in DB' });
+    const userService = new UserService(User);
```

Comment:
I think creating new service instance in every function is not the best approach. You could create one instance outside the functions or mark all UserService methods as static